.github/workflows/verifyAudit.yml

```diff
@@ -154,17 +162,23 @@ jobs:
               core.exportVariable('AUDIT_COMPLETED_ASSIGNED', 'true');
             }
             else
-              console.log(`Label 'AuditCompleted' is currently not assigned`);
+              console.log(`${colors.green}Label 'AuditCompleted' is currently not assigned${colors.reset}`);
+
+            // set CONTINUE to true to make sure the following steps are executed
+            core.exportVariable('CONTINUE', 'true');
```

Comment:
I notice that in most of these files, we’re using the shell-script approach to set the environment variable, like:
`echo "CONTINUE=true" >> "$GITHUB_ENV"`
It has the same outcome as using "core.exportVariable('CONTINUE', 'true')". 

IMO I prefer using `core.exportVariable` because it’s more readable approach but switching from it would mean updating code in a lot of places.

Also here: https://github.com/lifinance/contracts/pull/921/files#diff-a29fc6d92a576c3969d7e6f8fe90d37736a6528c50f8f427cd183c597b1db18eR162

Reply:
I would also lean towards using the explicit `core.exportVariable()` method. Seems less fragile.

Reply:
one is the typescript version and the other one is the bash version. 
In Bash code:  
```echo "CONTINUE=true" >> "$GITHUB_ENV"```

In javascript code:  
```"core.exportVariable('CONTINUE', 'true')"```

Reply:
@mirooon pls comment on or resolve the conversation